### ISSUE TITLE
Not working on Ubuntu 15.10

### ISSUE THREAD

**Issue by svanimpe** [elliottminns/uv-module#1]
Ubuntu installed `uv.h` in `/usr/include` instead of `/usr/local/include`.

**Comment by elliottminns** [comment]
How did you install libuv? Previously I had it with `libuv-dev` via apt-get but I realised this is an older version of libuv

The version that is supported can be installed via the below command:

```
$ sudo apt-get install automake libtool
$ curl -O http://dist.libuv.org/dist/v1.8.0/libuv-v1.8.0.tar.gz
$ tar xzf libuv-v1.8.0.tar.gz
$ cd libuv-v1.8.0 && sh autogen.sh
$ ./configure                                                       
$ make && sudo make install
```

**Comment by svanimpe** [comment]
Yes, I got it through `apt-get`, that's what I meant with "Ubuntu". It's version 1.6.1. Is that not recent enough? It does seem to work, other than the location of the header files.

**Comment by elliottminns** [comment]
When I was trying it with the `libuv-dev` version it appeared to have some missing features when I was compiling. I shall look into it more but currently I'm only guaranteeing with `1.8.0` :) 

Ideally, I want to move it into it's own Package once the Swift PM supports C projects as well.
